spec/controllers/v0/sign_in_controller_spec.rb

```diff
@@ -1385,9 +1497,9 @@
       end
       let(:logingov_id_token) { 'some-logingov-id-token' }
       let(:expected_status) { :ok }
-      let!(:user) { create(:user, :loa3, uuid: access_token_object.user_uuid, logingov_uuid: logingov_uuid) }
 
       before do
+        allow(User).to receive(:find).and_return(user)
```

Comment:
You shouldn't need this line since you `let!(:user)` above

Reply:
Same here.